index.html

```diff
@@ -240,58 +244,88 @@ <h2>Skills</h2>
           </li>
         </ul>
       </section>
-      <section class="get-in-touch">
-        <h3 class="intro-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
-<form action="https://formspree.io/f/mpzblelz" method="post">
-
-<ul>
-
-<li>
-<input type="text" required placeholder="First Name" maxlength="30" name="user_name">
-</li>
-<li>
-  <input required type="email" placeholder="Email address" name="user_email" >
-</li>
-<li>
-  <textarea name="user_message" maxlength="500" required id="message" cols="30" rows="10">Enter text here</textarea>
-</li>
-<li><button type="submit">Get in Touch</button></li>
-</ul>
+      <section class="contact-us get-in-touch">
+        <div class="intro-text">
+          <h3>
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h3>
```

Comment:
- [ ] As the previous reviewer pointed out, please make sure the font size for this heading matches the figma design in the desktop version. You can make it `font-size: 22px`